models/attachment.go

```diff
@@ -180,3 +226,20 @@ func DeleteAttachmentsByComment(commentID int64, remove bool) (int, error) {
 
 	return DeleteAttachments(attachments, remove)
 }
+
+// UpdateAttachment updates the given attachment in database
+func UpdateAttachment(atta *Attachment) error {
+	return updateAttachment(x, atta)
+}
+
+func updateAttachment(e Engine, atta *Attachment) error {
+	var sess *xorm.Session
+	if atta.ID != 0 || atta.UUID == "" {
+		sess = e.ID(atta.ID)
+	} else {
+		// Use uuid only if id is not set and uuid is set
+		sess = e.Where("uuid = ?", atta.UUID)
+	}
+	_, err := sess.AllCols().Update(atta)
```

Comment:
I don't think this is a good idea to update all the columns.

Reply:
What do you propose alternatively?

Reply:
```sess.Cols("col1", "col2"...).Update()```
Only update needed columns.